setup.cfg

```diff
@@ -33,10 +33,11 @@ install_requires =
     db-dtypes>=1.0.5
     google-cloud-bigquery>=3.4.0
     pandas>=1.3.4
-    pyarrow>=10.0.1
+    ; pyarrow>=10.0.1
```

Comment:
Why this ; ?

Reply:
??? no idea why there is this change·... I'll check. tnx to review.